schemas/api/fdc3UserInterfaceDrag.schema.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
I still don't think we're going to need this message.  Would be nice to merge my PoC into @julianna-ciq 's add-on module to prove that.

Reply:
The approach that you very briefly explained to positioning the iframe from inside by first resizing it to cover the whole parent window still seems risky, likely to cause display artefacts and is likely slower to apply (requires more messaging and rendering). 

We've also reviewed the current standard docs multiple times and checked the consensus of the SWG on them which we'd have to repeat if changing them significantly. We would need both a write-up and discussion of what you propose doing to change it and, personally, I don't think it worth further delaying this project to do so - no actual benefit to it has been articulated as yet.